### DISCUSSION TITLE
Amazon ADS, empty reports

### DISCUSSION THREAD

**Discussion by eduard-mg** [amzn/ads-advanced-tools-docs#343] [Q&A]
HI, I've been working on a program where I pull 3 kind of reports, Sponsored Brands, Sponsored Products and Sponsored Display using the Amazon ADS API (v3). The problem I'm having is that when all the campaigns on one of the profiles are either paused or archived, it returns an empty array ([]) and I would like to also see these campaigns. I think there might be some parameter I'm missing out on in the request. I tried using "state" with no results. Here is an example for Sponsored Brands (I'm using C#): 

...
  new Configuration
  {
      adProduct = "SPONSORED_BRANDS",
      groupBy = new List<string> {"campaign"},
      columns = new List<string> {"date", "newToBrandSalesClicks", "purchasesClicks", "detailPageViews" , "newToBrandSalesPercentage", "unitsSold", "sales", "qualifiedBorrowsFromClicks", "kindleEditionNormalizedPagesRoyalties14d", "addToListFromClicks", "costType", "campaignStatus", "royaltyQualifiedBorrowsFromClicks", "purchasesPromoted", "newToBrandDetailPageViewRate", "purchases", "campaignId", "newToBrandPurchasesPercentage", "kindleEditionNormalizedPagesRead14d", "qualifiedBorrows", "longTermROAS", "salesPromoted", "salesClicks", "newToBrandPurchasesRate", "unitsSoldClicks", "brandedSearches", "newToBrandECPDetailPageView", "royaltyQualifiedBorrows", "newToBrandPurchases", "longTermSales", "brandedSearchesClicks", "newToBrandSales", "addToList", "newToBrandPurchasesClicks", "addToCartClicks", "addToCart", "addToCartRate", "cost", "newToBrandDetailPageViewsClicks", "impressions", "newToBrandUnitsSold", "detailPageViewsClicks", "eCPAddToCart", "newToBrandDetailPageViews", "clicks", "newToBrandUnitsSoldClicks", "campaignName", "newToBrandUnitsSoldPercentage", "campaignRuleBasedBudgetAmount", "campaignBudgetCurrencyCode", "campaignBudgetType", "topOfSearchImpressionShare", "campaignBudgetAmount", "video5SecondViewRate", "video5SecondViews", "videoCompleteViews", "videoFirstQuartileViews", "videoMidpointViews", "videoThirdQuartileViews", "videoUnmutes", "viewabilityRate", "viewableImpressions", "viewClickThroughRate" },
      reportTypeId = "sbCampaigns",
      timeUnit = "DAILY",
      format = "GZIP_JSON"
  }, ...

...  
new CampaignRequest{
    name  = "SB_"+DateTime.Now.ToString("yyyyMMdd"), 
    startDate = startReport,
    endDate = endReport,
    configuration = configurations[0]
}, ...

**Comment by kdwivedi13** [comment]
For any troubleshooting needs, we encourage you to reach out to our dedicated support team. They can provide account-specific guidance and faster resolution: https://advertising.amazon.com/API/docs/en-us/support/overview

Best regards,
Kapil Dwivedi
